.gitlab/source_test/ebpf.yml

```diff
@@ -53,6 +53,7 @@
     paths:
       - $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files
 
+# mergequeue
```

Comment:
are those a left over todo or similar? I don't understand what the comment is about otherwise

Reply:
I had entered them originally to mark which jobs _should_ be running, but can remove them - it was easier than referring back to the required jobs settings.

Reply:
Addressed in [0afaafc](https://github.com/DataDog/datadog-agent/pull/20520/commits/0afaafcd3095f0e9ed78334e4438a2d7d0ecc46b)